tests/framework/caching/OciCacheTest.php

```
@@ -0,0 +1,74 @@
+{
+    private Connection|null $_connection = null;
+
+    protected function setUp(): void
+    {
+        if (!extension_loaded('pdo') || !extension_loaded('pdo_oci')) {
+            $this->markTestSkipped('pdo and pdo_oci extensions are required.');
+        }
+
+        $this->mockApplication();
+
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)